### ISSUE TITLE
Allow me to encrypt with the key I want regardless of email address

### ISSUE THREAD

**Issue by CalRobert** [mailvelope/mailvelope#476]
Mailvelope used to let me choose any of the public keys I had installed and target them accordingly. Now it insists they have to match what I've put in the "send" field, for reasons I can't understand. It also fails to realize when I've entered an address corresponding to an installed key. I realize this might have been done with good intentions but is very frustrating!

**Comment by lbort** [comment]
I second this. Additionally, it would be nice to be able to sign with multiple keys. I often send mails to an address, which is a "distribution list" or whatever the proper term is. What I mean is: Emails to group@example.com are forwarded to alice@example.com, bob@example.com and eve@example.com.

In my usecase, group@example.com is almost exclusively used for the communication by group members. So I am Bob, and want to send mails to everyone in the group. I have all the keys and know who is in the group. So I have to encrypt an email to group@example.com with alice's and eve's key (and additionally with my own, but this works already). 
There is no key for group@example.com, and there will never be one, because that would require sharing the secret key. Enigmail+thunderbird allows this behavior, it is even possible to configure it to do so automatically for all emails to group@example.com.

**Comment by robert-digit** [comment]
Similarly, the current approach makes it nearly impossible to send messages with keys that I got from keybase.

**Comment by s72785** [comment]
absolutely, the email field in key format is optional according to the standard, so there should be either an option to connect the imported keys to other addresses as well or at least possibility to select manually what key is wanted.

Also as stated before it eliminates other use cases, like the `group` option gpg uses or the rules Enigmail provides. A configuration like `pgprules.xml` of the latter maybe a good thing to build on top of, format wise.
